src/Symfony/Contracts/HttpClient/Test/TestHttpServer.php

```diff
@@ -43,4 +43,11 @@ public static function start(int $port = 8057)
 
         return $process;
     }
+
+    public static function stop(int $port = 8057)
+    {
```

Comment:
I usually use register_shutdown_function and it works pretty well 

https://github.com/jolicode/castor/blob/89eb467c6c9bc2a474f465dd3357cbd613d7fcd8/tests/Helper/WebServerHelper.php#L31

It could simplify the code